portal/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN npm cache clean --force
 COPY . .
 
 # Execute Script
-CMD ["npx", "ts-node", "server/index.ts"]
+CMD ["npx", "ts-node", "--transpile-only", "server/index.ts"]
```

Comment:
Why do we have this here and not elsewhere where we use TS out of curiousity

Reply:
We do this on production with the `TS_NODE_TRANSPILE_ONLY ` env var set to `true`, so this shouldn't be needed here. Leaving it out means we get an extra type check during dev deployments.

Reply:
The problem seems to come from `tsc` and `ts-node` being slightly different.
IDEs seem to use `tsc`'s type-checking which get the session types from the `express-session.d.ts` file.

As to why this isn't needed in `tfm-ui` I'm not sure

Reply:
It should work without this flag being explicitly used, so I don't think you should add it here. If we are going to add this it should be via the environment variable flag [here](https://typestrong.org/ts-node/docs/options/#transpileonly) :)